ksp_plugin_adapter/burn_editor.cs

```diff
@@ -56,29 +56,45 @@ class BurnEditor : ScalingRenderer {
     ComputeEngineCharacteristics();
   }
 
+  public enum Event {
+    None,
```

Comment:
`None` is mysterious.  `Unchanged`?

Reply:
It’s `Event.None`, which makes some sense; `Event.Unchanged` is weird.